src/index.html

```diff
@@ -17,6 +17,78 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo-moyo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo-moyo', a more descriptive alternative text like 'Moyo company logo' would be more informative.